azurelinuxagent/common/event.py

```diff
@@ -89,6 +89,7 @@ class WALAEventOperation:
     ExtensionHandlerManifest = "ExtensionHandlerManifest"
     ExtensionPolicy = "ExtensionPolicy"                                 # Event for any extension policy-related operations (e.g., extension not in allowlist).
     ExtensionSignaturePolicy = "ExtensionSignaturePolicy"               # Event for unsigned extension blocked due to extension signature policy.
+    ExtensionSigned = "ExtensionSigned"                                 # Event indicating whether an extension is signed.
```

Comment:
Alphabetized the list of telemetry events. I was looking for a particular event and the list was hard to read since it was not in alpabetical order.